Added new inventory file topics (#555)
RedHatInsights/red-hat-ansible-automation-platform-documentation: downstream/modules/platform/ref-sso-variables.adoc

[id="ref-sso-variables"]

= {RHSSO} variables

*Use these variables for `automationhub` or `automationcatalog`.

[cols="50%,50%",options="header"]
|====
| *Variable* | *Description* 

> | *`*_sso_console_admin_password`* | SSO admin password.

Reviewer: The asterisk before the variable name could be confusing: users might think that you should prepend the variables with  automationhub or automationcatalog.

| *`*_sso_console_keystore_file`* | Keystore file to install in SSO node.

`'/path/to/sso.jks'`
| *`*_sso_host`* | {CatalogNameStart} requires SSO and SSO admin credentials for
authentication. 

> SSO administration credentials are also required to set Catalog specific

Reviewer: ```suggestion
SSO administration credentials are also required to set {CatalogName} specific
```

roles needed for the application. 

> If SSO is not provided in the inventory for
> configuration, then the SSO host needs to be defined with this variable.

Reviewer: ```suggestion
If SSO is not provided in the inventory for
configuration, then you must use this variable to define the SSO host.
.
```
